pkgs/development/python-modules/wxPython/3.0.nix

```
@@ -37,7 +53,9 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ pkgconfig ]
     ++ (lib.optionals (!stdenv.isDarwin) [ wxGTK libX11 ])
-    ++ (lib.optionals stdenv.isDarwin [ wxmac darwin.apple_sdk.frameworks.Cocoa ]);
+    ++ (lib.optionals stdenv.isDarwin ([ wxmac ] ++ darwinFrameworks));
+
+  NIX_CFLAGS_COMPILE = lib.optionals stdenv.isDarwin (builtins.map (f: "-F${f}/Library/Frameworks") darwinFrameworks);
```

```suggestion
  buildInputs = []
    ++ lib.optionals (!stdenv.isDarwin) [ wxGTK.gtk ]
    ++ lib.optionals stdenv.isDarwin [
      Cocoa Carbon CoreServices DiskArbitration IOKit CFNetwork Security
      ApplicationServices CoreText CoreGraphics ImageIO AudioToolbox OpenGL
    ]
    ++ lib.optional openglSupport pyopengl;
```

This happens automatically for build inputs.

---

Ah.. so it was all because they were `nativeBuildInputs`

---

Yeah, python packages use `strictDeps` which makes the distinction important. For other packages you'd only run into this kind of problems when cross compiling.